person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require_relative 'Nameable'
+
```

Comment:
- [ ] It would be nice if you put nameable with a small `n` instead of Capital `N` because you have a file with the small **n** which is `nameable.rb` and also it is giving me error in the console 😨 Kindly fix it to make your app work as expected 😊

Reply:
@HAMMAS-SALEEM Done